### PR TITLE
Add more content to debug console

### PR DESCRIPTION
**Pull request by sdesen**
# Old Version

# New version

### CODE REVIEW

File: wiki0/src/client/components/home.tsx [hunk 1]
```diff
@@ -13,66 +16,164 @@ type AuthTokenType = {
...
-  const jsonPayload = decodeURIComponent(
-    window
-      .atob(base64)
-      .split('')
-      .map((c) => `%${`00${c.charCodeAt(0).toString(16)}`.slice(-2)}`)
-      .join('')
-  );
+const debugIDPEndpoint = (tokens: Record<string, any>[]) => {
+  const defaultUrl = 'IDP Token Endpoint';
+  if (!tokens.length || !tokens[0].jagToken) {
```
Comment (kamronbatman) [inline wiki0/src/client/components/home.tsx:21]: nit: collapse
```suggestion
  if (!tokens[0]?.jagToken) {
```

File: wiki0/src/client/components/DebugDrawer.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ const DebugDrawer: FC<PropsWithChildren<{ id: string }>> = function ({ id, child
           aria-controls={id}
           aria-label="Open Debug Console"
         >
-          <HiPlus />
+          <HiChevronLeft />
```
[the remark this answers is not in the document]
Comment (neelav) [reply wiki0/src/client/components/DebugDrawer.tsx:23]: screenshots in PR please

File: wiki0/src/client/components/tokenViewer.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import { Popover } from 'flowbite-react';
+import { FaRegCopy } from 'react-icons/fa';
+import { parseJwt } from '../utils';
+
+type TokenViewerProps = {
+  token: string | undefined | null;
+};
+
+function TokenViewer({ token }: TokenViewerProps) {
```
Comment (neelav) [inline wiki0/src/client/components/tokenViewer.tsx:9]: nice!

**Review by neelav — COMMENTED** [review]
lgtm, just please add screenshots to the PR for posterity

**Review by neelav — APPROVED** [review]

**Pull request merged**